### PR TITLE
Release process documentation

### PR DESCRIPTION
**Pull request by kanika1894**
Documentation stating the steps to release a new version of the extension.

### CODE REVIEW

File: ReleaseProcess.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Process for releasing a new version of VSCode extension - Deploy to Azure**
+
+1. Update the new version in package.json and package-lock.json 
+2. Add one liner for the new version in ChangeLog.md and make changes in readMe(if required).
+3. Create a release branch with the version name. Example, for version v1.2.3, create a new branch releases/v1.2.3.
+4. Create a tag for the corresponding version.
+5. Run the [pipeline](https://dev.azure.com/mseng/AzureDevOps/_build?definitionId=9571&_a=summary) with the release branch created.
+6. Download the artifacts and do one basic round of testing with the VSIX.
```
Comment (vineetmimrot) [inline ReleaseProcess.md:8]: Should we add "get the BVT tested by vendor's"?

File: ReleaseProcess.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+**Process for releasing a new version of VSCode extension - Deploy to Azure**
+
+1. Update the new version in package.json and package-lock.json 
+2. Add one liner for the new version in ChangeLog.md and make changes in readMe(if required).
```
Comment (vineetmimrot) [inline ReleaseProcess.md:4]: *nit: README

File: ReleaseProcess.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+3. Create a release branch with the version name. Example, for version v1.2.3, create a new branch releases/v1.2.3.
+4. Create a tag for the corresponding version.
```
Comment (vineetmimrot) [inline ReleaseProcess.md]: Shall we give a link, where to create tags.

**Review by vineetmimrot — APPROVED** [review]

File: ReleaseProcess.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+**Process for releasing a new version of VSCode extension - Deploy to Azure**
+
+1. Update the new version in package.json and package-lock.json 
+2. Add one liner for the new version in ChangeLog.md and make changes in README.(if required).
```
Comment (bishal-pdMSFT) [inline ReleaseProcess.md:4]: This need not be one liner. You can be more lines but should concisely tell what changed at a very high level

File: ReleaseProcess.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+3. Create a release branch with the version name. Example, for version v1.2.3, create a new branch releases/v1.2.3.
```
Comment (bishal-pdMSFT) [inline ReleaseProcess.md:5]: create a new release branch only for major version. For example, there should be new branch for v1, v2 etc

**Comment by bishal-pdMSFT** [comment]
Why delete README

**Comment by kanika1894** [comment]
> Why delete README
Readme isn't deleted.
There was an old doc named Release.md which isn't required anymore. That is deleted.

**Pull request merged**